src/entry-points/background/migrations/migrateFrom1_27_5.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * @license
+ * Copyright (C) 2022  WofWca <wofwca@protonmail.com>
+ *
+ * This file is part of Jump Cutter Browser Extension.
+ *
+ * Jump Cutter Browser Extension is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Jump Cutter Browser Extension is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with Jump Cutter Browser Extension.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+import { setSettings } from "@/settings";
```

Comment:
Also if you look at other migrations, you'll notice that they don't use `setSettings` because we might update `setSettings` to do a different thing, but a migration should never change once written. So please rewrite it with `browserOrChrome.storage.local.set`, as in the 1.25.1 migration.

Reply:
Perhaps it's a bit tinfoily, but this should not be too hard.